packages/catalog-process/src/services/catalogService.ts

```diff
@@ -938,9 +938,15 @@ export async function deleteDraftDescriptorLogic({
     throw notValidDescriptor(descriptorId, descriptor.state.toString());
   }
 
-  const interfacePath = descriptor.interface;
-  if (interfacePath !== undefined) {
-    await deleteFile(config.storageContainer, interfacePath.path);
+  const descriptorInterface = descriptor.interface;
+  if (descriptorInterface !== undefined) {
+    await deleteFile(config.storageContainer, descriptorInterface.path).catch(
+      (error) => {
+        logger.error(
+          `Error deleting interface for descriptor ${descriptorId} : ${error}`
+        );
+      }
```

Comment:
I think the error should be catched inside `deleteFile` when it will be implemented, for testing purposes we can leave it here but I suggest adding a comment that it has to be removed when `deleteFile` is implemented

Reply:
File manager is being implemented [here](https://github.com/pagopa/interop-be-monorepo/pull/198). It seems error catching is not included, so it should be handled when calling the functions.

Reply:
@taglioni-r about this topic: I was considering whether to add error catching or not in a further PR (#214) where I test all file manager calls in `catalog-process` - and opened the discussion on Slack: https://buildo.slack.com/archives/C0527LU1K4G/p1708338319303629?thread_ts=1707998126.299379&cid=C0527LU1K4G

Based on what we are saying in the Slack thread, if we want to keep the same behavior that Scala implements, we should not catch file manager call errors and, at the moment, they should result in `500` server errors. 
So, we could catch & log as you are doing but then rethrow the exception. What do you think?

(Then, as you suggested, we could track somewhere this future improvement and return a more meaningful error instead of `500`)

Reply:
Yes, I am following the thread.
I didn't realize I wasn't re-throwing the error, thanks!